tests/daikon-tests/purity/purity.txt-jml.goal

```diff
@@ -3,10 +3,10 @@ misc.Purity:::OBJECT
 this.value == this.getValue()
 this.shift == this.getShift()
-this.shift == this.retrieve()
-this.shift == this.retrieve()
+this.shift == this.retrieve(this.list1)
```

Comment:
What caused this change?  If it's correct, was this a bug fix unrelated to the exceptional exit support, but in the same pull request?  Or was it caused by a previously-committed change, but the goal files were not regenerated since then?

Reply:
Looking at the history of this goal file, it appears this is a new change.  I do not know what caused it.